README.rst

```diff
@@ -49,7 +49,7 @@ To also install all bears for coala at once, run:
 
 Be sure to use the latest version of pip, the default pip from Debian doesn't
 support our dependency version number specifiers. You will have to `use a
-virtualenv <https://github.com/coala/coala/wiki/FAQ#installation-is-failing-help>`__
+virtualenv <http://docs.coala.io/en/latest/Help/FAQ.html#installation-is-failing-help>`__
```

Comment:
Shouldn't it be `https` instead of `http`.
on hovering over the paragraph link, at the bottom of page it shows `https`. In general also `https` is preferable than `http`.